website/client/components/groups/questDetailsModal.vue

```diff
@@ -198,6 +198,14 @@ export default {
     },
   },
   methods: {
+    async questConfirm () {
+      let count = 0;
+      for (let uuid in this.group.quest.members) {
+        if (this.group.quest.members[uuid]) count += 1;
+      }
+      if (!confirm(this.$t('questConfirm', { questmembers: count, totalmembers: this.group.memberCount}))) return;
```

Comment:
We'd like to avoid using `confirm()` if possible; see https://github.com/HabitRPG/habitica/issues/9211. Would it be possible to have this pop out a stacked mini modal instead?

Reply:
Yup, I will work on this. Thanks for the reply.